packages/lesswrong/server/useractivities/cron.ts

```diff
@@ -0,0 +1,290 @@
+/* eslint-disable no-console */
```

Comment:
There are a lot of logs in this file, I would like to leave them in initially for debugging purposes. It only runs every 3 hours so I think this is fine